data/campaigns/Sceptre_of_Fire/scenarios/3_Searching_for_the_Runecrafter.cfg

```diff
@@ -372,24 +372,10 @@
                 carryover_percentage=40
             [/gold_carryover]
         [/objectives]
```

Comment:
Another thing which could be changed would be to drop the whole [objectives] block, and to add the one new one in the first one, hidden with an [show_if]. It has been changed that way in many campaigns, but not in all yet.